File: Starters/Starters 50/NEARESTCOURT/Solution.cpp
#include <ssctdlib>
#include <iostream>

int main() {
    int t,i,x,y,dif;
    cin>>t;
    for(i=0;i<t;i++){
        cin>>x>>y;
        dif = abs(x-y);
        if(dif%2==0){
            cout<<dif/2<<"\n";
        }
        else{
            cout<<(dif/2)+1<<"\n";
        }
    }
	return 0;
}

File: Starters/Starters 50/NEARESTCOURT/Solution.go
package main
import "fmt"
import "math"

func main(){
  var t,x,y,dif int
  fmt.Scanln(&t)
  for i:=0;i<t;i++ {
    fmt.Scan(&x,&y)
    dif = Abs(x-y)
    if dif%2 == 0 {
      fmt.Println(dif/2)
    } else {
      fmt.Println((dif/2)+1)
    }
  }
}
